### DISCUSSION TITLE
The chekpoint problem

### DISCUSSION THREAD

**Discussion by Junran5016** [MkuuWaUjinga/leopart#8] [Q&A]
When I set train_voc_config.yml, I set pretrained_weights as 'ours' ,and I set checkpoint as 'leopart_vits16.ckpt', only_load_weights is True.
But, there's a keyError: 'state_dict'. I don't know how to fix this. Is there anything I set wrong?
The below is the setting about finetune_with_leopart.py:
  @click.option("--config_path", default='configs/train_voc_config.yml', type=str)
  @click.option("--seed", type=int, default=400)
I still don't know how to fix this.
The leopart_vits16.ckpt is from README.md ViT-S/16 [full ckpt].
